solr/solr-ref-guide/src/index.adoc

```diff
@@ -90,6 +90,27 @@ The *<<getting-started.adoc#getting-started,Getting Started>>* section guides yo
 ****
 --
 
+[.row.match-my-cols]
```

Comment:
IMO this should not be a top-level topic. It's not broad enough. It's mostly about indexing nested documents, so I would argue it should be a child of indexing-and-basic-data-operations.adoc.

Reply:
Most of the additions concern with searching for nested documents.
Would it still fit under indexing-and-basic-data-operations.adoc?

Reply:
@ctargett thanks so much for your review!

It was my idea to have one page to get a gist of how to both index & query.  Admittedly such a page is hard to place in our TOC.  If pressed, I agree that if it had to go under either index or search sections, then index feels better because nested documents is more fundamentally data modeling.  @ctargett do you feel that moving this one page is fine or should we not attempt to consolidate the topic?  Even our consolidation is partial; we've left more details out in the search sections.

Reply:
A single page is fine IMO, even though it does make finding the right place for it more difficult. I think it's good to cover the indexing and the searching together when they are so specifically connected, as in this case.

If the search section can be improved with better context for the links that go out to other pages that cover the query parsers, then the bulk of the new content for the page would be about the data modelling needed to index nested documents, which makes a stronger argument for it being in the indexing section. But I'm not 100% on that opinion - if you guys would prefer it under Searching, that would be OK (someday I'd like to re-org the top-level topics, but that's another whole problem we can't solve here).